source/fundamentals/collations.txt

```diff
@@ -0,0 +1,531 @@
+       | :java-docs:`collationStrength() <apidocs/mongodb-driver-core/com/mongodb/client/model/Collation.Builder.html#collationStrength(com.mongodb.client.model.CollationStrength)>` API documentation
+
+   * - Locale
+     - | **Required**. The ICU locale code for language and variant.
+       | :java-docs:`locale() <apidocs/mongodb-driver-core/com/mongodb/client/model/Collation.Builder.html#locale(java.lang.String)>` API documentation
+
+   * - Normalization
+     - | Whether to perform unicode normalization on the text as needed. For more information on unicode normalization, see `Unicode Normalization Forms <https://unicode.org/reports/tr15/>`__.
+       | :java-docs:`normalization() <apidocs/mongodb-driver-core/com/mongodb/client/model/Collation.Builder.html#normalization(java.lang.Boolean)>` API documentation
+
+   * - Numeric Ordering
+     - | Whether to order numbers according to numeric value rather than collation order.
+       | :java-docs:`numericOrdering() <apidocs/mongodb-driver-core/com/mongodb/client/model/Collation.Builder.html#numericOrdering(java.lang.Boolean)>` API documentation
+
+You can use the ``Collation.Builder`` class to specify values for the
+collation options above. You can call the ``build()`` method construct a
+``Collation`` object as shown in the following code snippet:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CollationCollectionExample.java
+   :language: java
+   :dedent:
+   :start-after: start collationBuilder
+   :end-before: end collationBuilder
+
+For more information on the corresponding methods and parameters they
+take, see the API documentation for :java-docs:`Collation.Builder
+<apidocs/mongodb-driver-core/com/mongodb/client/model/Collation.Builder.html>`.
+
+
+.. _collations-examples:
+
+Collation Examples
+------------------
+
+This section contains examples that demonstrate how to use a selection of
+MongoDB operations that support collations. For each example, assume that
+you start with the following collection of documents:
+
+.. _collation-example-collection:
+
+.. code-block:: json
+
+  { "_id" : 1, "first_name" : "Klara" }
+  { "_id" : 2, "first_name" : "Gunter" }
+  { "_id" : 3, "first_name" : "Günter" }
+  { "_id" : 4, "first_name" : "Jürgen" }
+  { "_id" : 5, "first_name" : "Hannah" }
+
+In the following examples, we specify the "de@collation=phonebook" locale and
+variant collation. This contains rules for prioritizing proper nouns,
+identified by capitalization of the first letter. In an ascending order
+sort, the collation also orders characters with umlauts before the same
+characters without them.
+
+find() and sort() Example
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example demonstrates how you can apply a collation when
+retrieving sorted results from the sample collection. To perform this
+operation, call ``find()`` on the example collection and chain the
```

Comment:
Seems like it could be useful in a future discussion on terminology as "sample" and "example" seem to be used in our documentation.
Since it's within the same paragraph, I'm going to trim the first sentence since it doesn't need to be specific about the collection and can mislead the user into believing the code only works for the example collection.

Reply:
Added a stance on this issue to the [writing hints and tricks doc](https://docs.google.com/document/d/1SxlDvx4MYKBE6-LYGp5i5uuHfY1ZUYYlhc6syEdta5M/edit) as point 20 with a TODO for further discussion.